app/services/framework_nomis_mappings/personal_care_needs.rb

```diff
@@ -16,9 +17,15 @@ def call
   private
 
     def imported_personal_care_needs
-      @imported_personal_care_needs ||= NomisClient::PersonalCareNeeds.get(nomis_offender_numbers: [prison_number], personal_care_types: PERSONAL_CARE_NEED_CODES)
+      @imported_personal_care_needs ||= begin
+        personal_care_needs = NomisClient::PersonalCareNeeds.get(nomis_offender_numbers: [prison_number], personal_care_types: PERSONAL_CARE_NEED_CODES)
```

Comment:
You could use the same `.tap {}` pattern here if you want, but by no means essential and could be done later if you prefer.